packages/backend-dev-utils/src/ipcClient.ts

```diff
@@ -82,7 +94,7 @@ export class BackstageIpcClient {
 
       let timeout: NodeJS.Timeout | undefined = undefined;
 
-      const messageHandler = (response: Response) => {
+      const responseHandler: ResponseHandler = (response: Response) => {
         if (response?.type !== responseType) {
           return;
         }
```

Comment:
If we check the type in the listener, we can skip this

Reply:
Actually we can also skip the id check because of the map